tidb-in-kubernetes: add keepalive content in k8s troubleshooting (#1554)
pingcap/docs: dev/tidb-in-kubernetes/troubleshoot.md

        ```

    After the Pod is re-created, a new store is registered in the TiKV cluster. Then the recovery is completed.

## Long queries are abnormally interrupted in TiDB

> Load Balancers set the idle connection timeout. If no data is sent over a connection for a specific period of time, Load Balancer closes the connection.

Reviewer: ```suggestion
Load balancers often set the idle connection timeout. If no data is sent over a connection for a specific period of time, the load balancer closes the connection.
```

> If a long query is interrupted when you use TiDB, check the middleware application between the client and the TiDB server.

Reviewer: ```suggestion
If a long query is interrupted when you use TiDB, check the middleware program between the client and the TiDB server.
```

> If you find that the idle timeout is not long enough for your query, try to set the timeout to a larger value. If you cannot reset it, enable the `tcp-keep-alive` option.

Reviewer: ```suggestion
If the idle timeout is not long enough for your query, try to set the timeout to a larger value. If you cannot reset it, enable the `tcp-keep-alive` option in TiDB.
```

> In Linux, the keepalive packets are sent at the interval of 7,200 seconds by default. To shorten the interval, configure `sysctls` via the `podSecurityContext` field. Here is an example:

Reviewer 2: The `tcp_keepalive_time` interval is not between keepalive packets. It is between the first probe packet and the last data packet. 

The first keepalive probe packet is sent after 7,200 seconds since the last data packet by default.

> In Linux, the first keepalive probe packet is sent after 7,200 seconds since the last data packet by default. To shorten the interval, configure `sysctls` via the `podSecurityContext` field. Here is an example:

Reviewer: ```suggestion
In Linux, the keepalive probe packet is sent every 7,200 seconds by default. To shorten the interval, configure `sysctls` via the `podSecurityContext` field. Here is an example:
```
